src/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs

```diff
@@ -412,6 +412,7 @@ public void UrlDecode(string decoded, string encoded)
 
         [Theory]
         [MemberData(nameof(UrlDecodeData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Full framework assumes the current codepage is UTF8")]
```

Comment:
Yes I looked at it and it looked weird to me as well.

It seems like an issue in `HttpUtility.UrlDecodeToBytes` as `HttpUtility.UrlDecode` test is not failing with the same input, so I will go ahead and open an issue to track this. Thanks @ianhays